fenix/app/src/test/java/org/mozilla/fenix/shopping/fake/FakeReviewQualityCheckService.kt

```diff
@@ -30,5 +32,14 @@ class FakeReviewQualityCheckService(
     override suspend fun analysisStatus(): AnalysisStatusDto? = status
 
     override fun selectedTabUrl(): String? = selectedTabUrl
+
     override suspend fun productRecommendation(): ProductRecommendation? = productRecommendation
+
+    override suspend fun recordRecommendedProductClick(productAid: String) {
+        recordClick(productAid)
+    }
```

Comment:
super curious if this will work
```suggestion
    override suspend fun recordRecommendedProductClick(productAid: String) = recordClick
```

Reply:
😄 doesn't work, `recordClick` is of type  `(String) -> Unit`, and this function returns a `Unit`